src/main/java/ch/njol/skript/expressions/ExprBlocks.java

```diff
@@ -163,10 +185,15 @@ public boolean isSingle() {
 	
 	@Override
 	public String toString(final @Nullable Event e, final boolean debug) {
-		final Expression<Block> end = this.end;
-		if (end != null) {
+		final Expression<Location> end = this.end;
+		if (chunk != null)

```

Comment:
Nitpick but can you put curly braces on all the if / else if statements here ;D

Reply:
done 👍